### PR TITLE
[TRITON] GEMM kernels nomenclature changes

### PR DESCRIPTION
**Pull request by Boss2002n**
Uses the helper from https://github.com/ROCm/aiter/pull/1260 to add config-aware naming to kernels

- Applied repr to 7 MOE kernel files 
- Applied repr to prefill attention kernels
- Applied repr to softmax and topk kernels

Examples:
Before: _fwd_kernel
After: _fwd_kernel_kv_group_num_8_BLOCK_M_128_BLOCK_DMODEL_128_BLOCK_N_64_IS_CAUSAL_1_Lk_4096

Before: _fused_moe_persistent_kernel_gptq_awq
After: _fused_moe_persistent_kernel_gptq_awq_N_14336_K_4096_group_size_128_BLOCK_SIZE_M_64_BLOCK_SIZE_N_128_BLOCK_SIZE_K_128_GROUP_SIZE_M_8_EVEN_K_1_MUL_ROUTED_WEIGHT_1_top_k_2_compute_type_FP16_has_zp_0_use_int4_w4a16_1_use_int8_w8a16_0_NUM_XCDS_8

### CODE REVIEW

**Review by lucas-santos-amd — CHANGES_REQUESTED** [review]
Looks great! While you are at it, can please remove the commented code and the parameters you marked as "not being used in the kernel"?

File: aiter/ops/triton/_triton_kernels/extend_attention.py [hunk 1]
```diff
@@ -31,9 +31,33 @@
...
+        "BLOCK_DV",
+        "BLOCK_M",
+        "BLOCK_N",
+        "USE_CUSTOM_MASK",
+        "IS_CAUSAL",
+        "SKIP_PREFIX_CUSTOM_MASK",
+        "STORE_TRANSPOSE",
+        "NUM_Q_HEADS",
+        "NUM_BLOCKS",
+        # "BATCH",
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/_triton_kernels/extend_attention.py]: Here

File: aiter/ops/triton/_triton_kernels/extend_attention.py [hunk 2]
```diff
@@ -74,7 +98,7 @@ def _fwd_kernel(
     STORE_TRANSPOSE: tl.constexpr,
     NUM_Q_HEADS: tl.constexpr,
     NUM_BLOCKS: tl.constexpr,
-    BATCH: tl.constexpr,
+    # BATCH: tl.constexpr,
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/_triton_kernels/extend_attention.py]: Here

File: aiter/ops/triton/extend_attention.py [hunk 3]
```diff
@@ -20,7 +20,7 @@
 from typing import Optional
 import torch
 import triton
-import triton.language as tl
+# import triton.language as tl
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/extend_attention.py]: Here

File: aiter/ops/triton/extend_attention.py [hunk 4]
```diff
@@ -133,7 +154,7 @@ def extend_attention_fwd(
         STORE_TRANSPOSE=True,
         NUM_Q_HEADS=head_num,
         NUM_BLOCKS=num_blocks,
-        BATCH=batch_size,
+        # BATCH=batch_size,
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/extend_attention.py]: Here

File: aiter/ops/triton/moe_align_block_size.py [hunk 5]
```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2024-2025, Advanced Micro Devices, Inc. All rights reserved.
 
 import torch
-import triton
-import triton.language as tl
+# import triton
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_align_block_size.py]: Here

File: aiter/ops/triton/moe_op.py [hunk 6]
```diff
@@ -143,7 +168,7 @@ def fused_moe(
                 num_tokens_post_padded,
                 B.shape[1],
                 A.shape[1],
-                EM,
+                EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op.py]: Here

File: aiter/ops/triton/moe_op.py [hunk 7]
```diff
@@ -186,7 +211,7 @@ def fused_moe(
...
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op.py:213]: Here

File: aiter/ops/triton/moe_op.py [hunk 8]
```diff
@@ -236,7 +261,7 @@ def fused_moe(
                 num_tokens_post_padded,
                 B.shape[1],
                 A.shape[1] - _PADDING_SIZE,
-                sorted_token_ids.shape[0],
+                sorted_token_ids.shape[0],  # (EM) it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op.py]: Here

File: aiter/ops/triton/moe_op.py [hunk 9]
```diff
@@ -278,7 +303,7 @@ def fused_moe(
                 num_tokens_post_padded,
                 B.shape[1],
                 A.shape[1] - _PADDING_SIZE,
-                EM,
+                EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op.py]: Here

File: aiter/ops/triton/moe_op_e2e.py [hunk 10]
```diff
@@ -3,7 +3,7 @@
 
 import torch
 import triton
-import triton.language as tl
+# import triton.language as tl
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_e2e.py]: Here

File: aiter/ops/triton/moe_op_gelu.py [hunk 11]
```diff
@@ -131,7 +154,7 @@ def fused_moe_gelu(
             num_tokens_post_padded,
             B.shape[1],
             A.shape[1] - _PADDING_SIZE,
-            sorted_token_ids.shape[0],
+            sorted_token_ids.shape[0],  # (EM) it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_gelu.py]: Here

File: aiter/ops/triton/moe_op_gelu.py [hunk 12]
```diff
@@ -174,7 +197,7 @@ def fused_moe_gelu(
             num_tokens_post_padded,
             B.shape[1],
             A.shape[1] - _PADDING_SIZE,
-            EM,
+            EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_gelu.py]: Here

File: aiter/ops/triton/moe_op_mxfp4.py [hunk 13]
```diff
@@ -92,7 +116,7 @@ def fused_moe_mxfp4(
         num_tokens_post_padded,
         B.shape[1],
         A.shape[1],
-        EM,
+        EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_mxfp4.py]: Here

File: aiter/ops/triton/moe_op_mxfp4_silu_fused.py [hunk 14]
```diff
@@ -91,7 +115,7 @@ def fused_moe_mxfp4_silu(
         num_tokens_post_padded,
         B.shape[1],
         A.shape[1],
-        EM,
+        EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_mxfp4_silu_fused.py]: Here

File: aiter/ops/triton/moe_op_silu_fused.py [hunk 15]
```diff
@@ -141,7 +166,7 @@ def fused_moe_silu(
                 num_tokens_post_padded,
                 B.shape[1],
                 A.shape[1],
-                EM,
+                EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_silu_fused.py]: Here

File: aiter/ops/triton/moe_op_silu_fused.py [hunk 16]
```diff
@@ -185,7 +210,7 @@ def fused_moe_silu(
...
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_silu_fused.py]: Here

File: aiter/ops/triton/moe_op_silu_fused.py [hunk 17]
```diff
@@ -277,7 +302,7 @@ def fused_moe_silu(
                 num_tokens_post_padded,
                 B.shape[1],
                 A.shape[1] - _PADDING_SIZE,
-                EM,
+                EM,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_silu_fused.py]: Here

File: aiter/ops/triton/moe_op_silu_fused.py [hunk 18]
```diff
@@ -235,7 +260,7 @@ def fused_moe_silu(
                 num_tokens_post_padded,
                 B.shape[1],
                 A.shape[1] - _PADDING_SIZE,
-                sorted_token_ids.shape[0],
+                sorted_token_ids.shape[0],  # (EM) it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_op_silu_fused.py]: Here

File: aiter/ops/triton/moe_routing_sigmoid_top1_fused.py [hunk 19]
```diff
@@ -4,7 +4,7 @@
 from typing import Optional
 import torch
 import triton
-import triton.language as tl
+# import triton.language as tl
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/moe_routing_sigmoid_top1_fused.py]: Here

File: aiter/ops/triton/prefill_attention.py [hunk 20]
```diff
@@ -23,7 +23,7 @@
 # Adapted from
 # https://github.com/ModelTC/lightllm/blob/f2a54f0912293f683bf1d1695fd12c4098a5bf82/lightllm/models/llama/triton_kernel/context_flashattention_nopad.py#L1
 import triton
-import triton.language as tl
+# import triton.language as tl
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/prefill_attention.py]: Here

File: aiter/ops/triton/softmax.py [hunk 21]
```diff
@@ -1,6 +1,6 @@
 import torch
 import triton
-import triton.language as tl
+# import triton.language as tl
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/softmax.py]: Here

File: aiter/ops/triton/softmax.py [hunk 22]
```diff
@@ -40,7 +36,7 @@ def softmax(x):
         x,
         x.stride(0),
         y.stride(0),
-        n_rows,
+        n_rows,  # it's not being used in the kernel
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/softmax.py]: Here

File: aiter/ops/triton/topk.py [hunk 23]
```diff
@@ -11,8 +11,9 @@
 import torch
 import triton
 import triton.language as tl
-import triton.language.core as core
-from triton.language.standard import _log2, zeros_like
+
+# import triton.language.core as core
```
Comment (lucas-santos-amd) [inline aiter/ops/triton/topk.py]: Here

**Review by lucas-santos-amd — APPROVED** [review]
